cmd/file_kong2kic.go

```diff
@@ -0,0 +1,139 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/kong/deck/kong2kic"
+	"github.com/kong/go-apiops/logbasics"
+	"github.com/kong/go-database-reconciler/pkg/file"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdKong2KicInputFilename  string
+	cmdKong2KicOutputFilename string
+	cmdKong2KicOutputFormat   string
+	cmdKong2KicClassName      string
+	cmdKong2KicIngress        bool
+	cmdKong2KicVersion        string
+)
+
+// Executes the CLI command "kong2kic"
+func executeKong2Kic(cmd *cobra.Command, _ []string) error {
+	_ = sendAnalytics("file-kong2kic", "", modeLocal)
+	var (
+		outputContent    *file.Content
+		err              error
+		outputFileFormat file.Format
+		yamlOrJSON       string
+	)
+
+	kong2kic.ClassName = cmdKong2KicClassName
+	verbosity, _ := cmd.Flags().GetInt("verbose")
+	logbasics.Initialize(log.LstdFlags, verbosity)
+
+	// if cmdKong2KicVersion is not 2 or 3 return an error
+	if cmdKong2KicVersion != "2" && cmdKong2KicVersion != "3" {
```

Comment:
since you already have a `validateInput` helper function, it would make sense to consolidate all input validation there.